src/apps/clever/contracts/abis/clever-gauge.json

```diff
@@ -0,0 +1,562 @@
+[
+  {
+    "name": "Deposit",
+    "inputs": [
+      { "name": "provider", "type": "address", "indexed": true },
+      { "name": "value", "type": "uint256", "indexed": false }
+    ],
+    "anonymous": false,
+    "type": "event"
+  },
+  {
+    "name": "Withdraw",
+    "inputs": [
+      { "name": "provider", "type": "address", "indexed": true },
+      { "name": "value", "type": "uint256", "indexed": false }
+    ],
+    "anonymous": false,
+    "type": "event"
+  },
+  {
+    "name": "UpdateLiquidityLimit",
+    "inputs": [
+      { "name": "user", "type": "address", "indexed": false },
+      { "name": "original_balance", "type": "uint256", "indexed": false },
+      { "name": "original_supply", "type": "uint256", "indexed": false },
+      { "name": "working_balance", "type": "uint256", "indexed": false },
+      { "name": "working_supply", "type": "uint256", "indexed": false }
+    ],
+    "anonymous": false,
+    "type": "event"
+  },
+  {
+    "name": "CommitOwnership",
+    "inputs": [{ "name": "admin", "type": "address", "indexed": false }],
+    "anonymous": false,
+    "type": "event"
+  },
+  {
+    "name": "ApplyOwnership",
+    "inputs": [{ "name": "admin", "type": "address", "indexed": false }],
+    "anonymous": false,
+    "type": "event"
+  },
+  {
+    "name": "Transfer",
+    "inputs": [
+      { "name": "_from", "type": "address", "indexed": true },
+      { "name": "_to", "type": "address", "indexed": true },
+      { "name": "_value", "type": "uint256", "indexed": false }
+    ],
+    "anonymous": false,
+    "type": "event"
+  },
+  {
+    "name": "Approval",
+    "inputs": [
+      { "name": "_owner", "type": "address", "indexed": true },
+      { "name": "_spender", "type": "address", "indexed": true },
+      { "name": "_value", "type": "uint256", "indexed": false }
+    ],
+    "anonymous": false,
+    "type": "event"
+  },
+  {
+    "stateMutability": "nonpayable",
+    "type": "constructor",
+    "inputs": [
+      { "name": "_lp_token", "type": "address" },
+      { "name": "_minter", "type": "address" },
+      { "name": "_admin", "type": "address" }
+    ],
+    "outputs": []
+  },
+  {
+    "stateMutability": "view",
+    "type": "function",
+    "name": "decimals",
+    "inputs": [],
+    "outputs": [{ "name": "", "type": "uint256" }],
+    "gas": 360
```

Comment:
Remove all "gas" properties within the .json files, then delete /ethers folder + index and regenerate the contracts will resolve the issue

Reply:
Done!